src/modules/collection/utils.spec.ts

```diff
@@ -239,3 +240,82 @@ describe('when transforming a CollectionPaginationData to PaginationStats', () =
     })
   })
 })
+
+describe.only('when getting the fiat commodity amount', () => {
```

Comment:
Would you mind removing the only here?
```suggestion
describe('when getting the fiat commodity amount', () => {
```